crates/holochain_sqlite/src/sql/cell/schema.sql

```diff
@@ -128,6 +130,10 @@ CREATE TABLE IF NOT EXISTS DhtOp (
     FOREIGN KEY(header_hash) REFERENCES Header(hash)
 );
 CREATE INDEX IF NOT EXISTS DhtOp_type_idx ON DhtOp ( type );
+CREATE INDEX IF NOT EXISTS DhtOp_validation_stage_idx ON DhtOp ( validation_stage );
+CREATE INDEX IF NOT EXISTS DhtOp_validation_status_idx ON DhtOp ( validation_status );
+CREATE INDEX IF NOT EXISTS DhtOp_authored_timestamp_ms_idx ON DhtOp ( authored_timestamp_ms );
+CREATE INDEX IF NOT EXISTS DhtOp_storage_center_loc_idx ON DhtOp ( storage_center_loc );
```

Comment:
Did you find that separate individual indexes worked better? I was under the impression that it was more efficient to have a separate grouped index for the set of fields used by each query... so for

```sql
WHERE
    DhtOp.authored_timestamp_ms >= :from
    AND DhtOp.authored_timestamp_ms < :to
    AND storage_center_loc >= :storage_start_1
    AND storage_center_loc <= :storage_end_1
```

since it uses `authored_timestamp_ms` and `storage_center_loc`, you would want an index like:

```sql
CREATE INDEX IF NOT EXISTS DhtOp_fetch_op_hashes_single_idx
ON DhtOp (
    authored_timestamp_ms,
    storage_center_loc
);
```

Reply:
Yeh I tried both and it never uses the second index regardless of whether they were combined or not.
This is the query plan that I get with both indexes:
```
QUERY PLAN
`--COMPOUND QUERY
   |--LEFT-MOST SUBQUERY
   |  `--SEARCH TABLE DHtOp USING INDEX DhtOp_storage_center_loc_idx (storage_center_loc>? AND storage_center_loc<?)
   `--UNION ALL
      `--SEARCH TABLE DHtOp USING INDEX DhtOp_storage_center_loc_idx (storage_center_loc>? AND storage_center_loc<?)
```
It doesn't even mention the index for authored_timestamp_ms and this is the same if I combine them (I tried both combos)